bin/behat-prepare.sh

```diff
@@ -169,78 +145,59 @@ foreach (\$config['example-userpass'] as \$key => &\$user) {
-cp -r "$PREPARE_DIR"/private/simplesamlphp/config "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/templates "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/cert "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/metadata "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/routing "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/attributemap "$PREPARE_DIR"/simplesaml/
-cp -r "$PREPARE_DIR"/private/simplesamlphp/lib "$PREPARE_DIR"/simplesaml/
-
-# Modify the include...
+mkdir -p "$PREPARE_DIR/simplesaml"
+cp -r "$PREPARE_DIR/private/simplesamlphp/public/"*   "$PREPARE_DIR/simplesaml/"
+cp -r "$PREPARE_DIR/private/simplesamlphp/"{vendor,src,modules,config,templates,cert,metadata,routing,attributemap,lib} "$PREPARE_DIR/simplesaml/"
 sed -i "s|dirname(__FILE__, 2) . '/src/_autoload.php'|__DIR__ . '/src/_autoload.php'|" "$PREPARE_DIR/simplesaml/_include.php"
 
-###
-# Push files to the environment
-###
-git add private wp-content simplesaml
-git config user.email "wp-saml-auth@getpantheon.com"
-git config user.name "Pantheon"
-git commit -m "Include SimpleSAMLphp and its configuration files"
-git push
+git -C "$PREPARE_DIR" add private wp-content simplesaml
+git -C "$PREPARE_DIR" config user.email "wp-saml-auth@getpantheon.com"
+git -C "$PREPARE_DIR" config user.name "Pantheon"
+git -C "$PREPARE_DIR" commit -m "Include SimpleSAMLphp and its configuration files" || true
+git -C "$PREPARE_DIR" push
 
-# Sometimes Pantheon takes a little time to refresh the filesystem
-terminus build:workflow:wait "$SITE_ENV"
+# NOTE: some Terminus installs don't have build:workflow; use workflow:wait
+terminus workflow:wait "$SITE_ENV"
 
-###
-# Set up WordPress, theme, and plugins for the test run
-###
-# Silence output so as not to show the password.
 {
-  terminus wp "$SITE_ENV" -- core install --title="$TERMINUS_ENV"-"$TERMINUS_SITE" --url="$PANTHEON_SITE_URL" --admin_user="$WORDPRESS_ADMIN_USERNAME" --admin_email="$WORDPRESS_ADMIN_EMAIL" --admin_password="$WORDPRESS_ADMIN_PASSWORD"
-} &> /dev/null
-terminus wp "$SITE_ENV" -- option update home "https://$PANTHEON_SITE_URL"
+  terminus wp "$SITE_ENV" -- core install \
+    --title="${TERMINUS_ENV}-${TERMINUS_SITE}" \
+    --url="$PANTHEON_SITE_URL" \
+    --admin_user="$WORDPRESS_ADMIN_USERNAME" \
+    --admin_email="$WORDPRESS_ADMIN_EMAIL" \
+    --admin_password="$WORDPRESS_ADMIN_PASSWORD"
+} &> /dev/null || true
+
+terminus wp "$SITE_ENV" -- option update home   "https://$PANTHEON_SITE_URL"
 terminus wp "$SITE_ENV" -- option update siteurl "https://$PANTHEON_SITE_URL"
 terminus wp "$SITE_ENV" -- plugin activate wp-native-php-sessions wp-saml-auth
 terminus wp "$SITE_ENV" -- theme activate "$TERMINUS_SITE"
 terminus wp "$SITE_ENV" -- rewrite structure '/%year%/%monthnum%/%day%/%postname%/'
-# Create writeable directories in /files (aka wp-content/uploads) that SimpleSAMLphp might need.
```

Comment:
This comment should not be removed. This was a hack and this comment provides context on why it was necessary to do this way.